core/unix/signal_macos.c

```diff
@@ -157,8 +157,16 @@ sigcontext_to_mcontext_simd(priv_mcontext_t *mc, sig_full_cxt_t *sc_full)
         return;
     mc->fpsr = fpc->__fpsr;
     mc->fpcr = fpc->__fpcr;
-    for (int i = 0; i < proc_num_simd_registers(); i++) {
-        memcpy(&mc->simd[i].q, &fpc->__v[i], sizeof(mc->simd->q));
+    if (proc_has_feature(FEATURE_SVE)) {
+        /* XXX i#5383: SVE and SVE2 support for MACOS still missing.
+         */
+        ASSERT_NOT_IMPLEMENTED(false);
```

Comment:
Thanks!

Btw, when you're ready, you can resolve all conversation, merge with master (resolve conflicts if necessary), wait for all tests to run, and finally "squash and merge" this PR yourself.

Thank you for contributing!